index.html

```diff
@@ -198,6 +197,7 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
             <textarea required class="form-item " maxlength="500" id="msg" name="message_area"
                 placeholder="Enter test here"></textarea>
             <button>Get in touch</button>
+            <p class="error">the email should in the right format</p>
```

Comment:
- [ OPTIONAL]
 Your app works well and it looks great, however, it would be nice to display a more descriptive error message that tells the user what to do to solve the error. For example:

Your error message could be something like: 
> `You entered an incorrect email format, please enter your email with lowercase characters`

Reply:
Noted @julie-ify  👍